test/sandbox/vscode/.vscode/settings.json

```diff
@@ -8,6 +8,8 @@
             "command": [
                 "../../../bazel-bin/main/sorbet",
                 "--debug-log-file=sorbet-vscode.log",
+                "--ignore=things",
+                "--rubyfmt-path=/Users/trevor/stripe/rubyfmt/target/release/rubyfmt",
```

Comment:
These changes probably should not be committed?

Reply:
Definitely not, thank you!